### DISCUSSION TITLE
如何成为pcl2的github协作者

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#1721] [提问] [answered]
？

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #477 这里有详细的解释。按照经验和惯例概括就是：
- 账户较为活跃，且有 Minecraft 仓库（非必要）可以熟练使用 Github；
- 经常协助开发者及 Collaborator 处理 Issue，并可以提供有用信息以辅助 Issues 处理；
- 在 Discussions 中活跃且回答质量较高；
- 可以提供其他有助于该仓库管理的其他事项。

**Comment by iPodtouch6557** [comment]
参见 #63 和 #477
